src/main/java/com/welcome/bot/security/oauth2/CustomOAuth2UserService.java

```diff
@@ -58,7 +58,7 @@ private OAuth2User processOAuth2User(OAuth2UserRequest oAuth2UserRequest, OAuth2
     }
     
     private User updateExistingUser(User existingUser, OAuth2UserInfo oAuth2UserInfo) {
-        existingUser.setName(oAuth2UserInfo.getName().substring(0,oAuth2UserInfo.getName().indexOf(" ")));
+        existingUser.setName(oAuth2UserInfo.getName().substring(0,5));
```

Comment:
I do not understand this line. Can you explain it?

Reply:
Substring returns string of limited characters(I want to have user's first name only).

Reply:
Why limit number of characters? What about longer names?

Reply:
It goes to first space.

Reply:
That solution was before and it was not bad. Limiting number of chars can produce wrong results for most cases.

Reply:
Problem is with characters in our language like "č,ć,š,ž" that won't be saved in database. I know it's bad but it's working.